mkautodoc/extension.py

```diff
@@ -42,24 +42,47 @@ def get_params(signature: inspect.Signature) -> typing.List[str]:
     render_kw_only_separator = True
 
     for parameter in signature.parameters.values():
-        value = parameter.name
-        if parameter.default is not parameter.empty:
-            value = f"{value}={parameter.default!r}"
+        param = etree.Element("span", {"class": "autodoc-param-definition"})
 
+        value = parameter.name
         if parameter.kind is parameter.VAR_POSITIONAL:
             render_kw_only_separator = False
             value = f"*{value}"
         elif parameter.kind is parameter.VAR_KEYWORD:
             value = f"**{value}"
+        param_name = etree.SubElement(
+            param, "em", {"class": "autodoc-param autodoc-param-name"}
+        )
+        param_name.text = value
+
+        if parameter.annotation is not parameter.empty:
+            colon = etree.SubElement(param, "span", {"class": "autodoc-punctuation"})
+            colon.text = ": "
+            type = etree.SubElement(param, "span", {"class": "autodoc-type-annotation"})
```

Comment:
@dhirschfeld if you want different styling on type annotations, you would just have to taret this `autodoc-type-annotation` in your custom css. as far as i can tell, `mkautodoc` doesn't ship its own default styling

Reply:
e.g. for mkdocs you can do extra_css, or you can do custom theming stuff https://www.mkdocs.org/user-guide/styling-your-docs/#using-the-docs_dir

Reply:
Thanks for the pointer! I'll give it a go...